doc/api_assets/style.css

```diff
@@ -1,5 +1,6 @@
 /*--------------------- Layout and Typography ----------------------------*/
 html {
+  -webkit-overflow-scrolling: touch;
```

Comment:
Like what? I've tried it on my mac, and it doesn't seem to affect any of the browsers I've tested it on (Safari, Chrome, Firefox).